README.md

```diff
@@ -85,7 +85,7 @@ sudo apt install git curl freeglut3-dev autoconf libx11-dev \
     libglu1-mesa-dev libgles2-mesa-dev libegl1-mesa-dev \
     pulseaudio dbus-x11 libavcodec-dev libavformat-dev \
     libavutil-dev libswresample-dev  libswscale-dev libdbus-1-dev \
```

Comment:
https://github.com/servo/servo/pull/19356/files
Please revert according to this PR